README.md

```diff
@@ -0,0 +1,85 @@
+# Django Lightweight Queue
+
+DLQ is a lightweight & modular queue and cron system for Django.
+
+## Basic Usage
+
+``` python
+import time
+from django_lightweight_queue import task
+
+# Define a task
+@task()
+def long_running_task(first_arg, second_arg):
+    time.sleep(first_arg * second_arg)
+
+# Request that the task be executed at some point
+long_running_task(4, second_arg=9)
+```
+
+See the docstring on the [`task`](django_lightweight_queue/task.py) decorator
+for more details.
+
+## Configuration
+
+Configuration options should be placed in the usual Django settings files,
+prefixed with `LIGHTWEIGHT_QUEUE_`, for example:
+``` python
+LIGHTWEIGHT_QUEUE_BACKEND = 'django_lightweight_queue.backends.redis.RedisBackend'
+```
+
+### Backends
+
+There are three built-in backends:
+- Synchronous (the default): executes the task inline, without any actual queuing
+- Redis: executes tasks at-most-once using [Redis][redis] for storage of the
+  enqueued tasks
+- Reliable Redis: executes tasks at-least-once using [Redis][redis] for storage
+  of the enqueued tasks
+
+[redis]: https://redis.io/
+
+## Running Workers
+
+The queue runner is implemented as a Django management command:
+```
+$ python manage.py queue_runner
+```
+
+Workers can be shared over multiple hosts by telling each runner that it is part
```

Comment:
I think "distributed" rather than "shared" - or "spread" if you'd prefer to avoid technical language - would solve this.